Makefile

```diff
@@ -27,6 +29,13 @@ test: deps $(SOURCES)
 	PATH=$(CURRENT_DIR)/bin:$(PATH) go test $(PACKAGES) $(TESTARGS)
 	go vet $(PACKAGES)
 
+gometalint-exists:
+	@which gometalinter > /dev/null || \
```

Comment:
good idea, but i would rather not have makefile installing dependencies when running build like this. instead we can create new target like `install-tools` which would do this job, and error message in `-exists` tasks would point to running `make install-tools`. So i will leave this as it is for now and we can introduce this additional target along with changing travis.yaml to use Make